src/main/webapp/WEB-INF/jsp/add-deal.jsp

```diff
@@ -7,6 +7,7 @@
   <meta name="description" content="" />
   <meta name="author" content="" />
   <title>Add Deal</title>
+  <script src="/js/util.js"></script>
```

Comment:
Should this change be in a separate PR?

Reply:
the util.js file contains the authentication for the frontend side. It doesn't need to be in this PR, but I thought it was somewhat relevant.